submissions/antonkottans/hooli-style-popup/index.html

```diff
@@ -0,0 +1,143 @@
+                        <div class="more-apps-1">
+                            <hr class="popup-hr">
+                            <ul class="hidden-items-list">
+                                <li class="hidden-popup-list-item">
+                                    <a href="#" class="popup-item">
+                                        <img class="popup-item-img" alt="" src="img/chat.png">Chat
+                                    </a>
+                                </li>
+                                <li class="hidden-popup-list-item">
+                                    <a href="#" class="popup-item">
+                                        <img class="popup-item-img" alt="" src="img/classroom.png">Classroom
+                                    </a>
+                                </li>
+                                <li class="hidden-popup-list-item">
+                                    <a href="#" class="popup-item">
+                                        <img class="popup-item-img" alt="" src="img/collection-cut.png">Collection
+                                    </a>
+                                </li>
+                                <li class="hidden-popup-list-item">
+                                    <a href="#" class="popup-item">
+                                        <img class="popup-item-img" alt="" src="img/mars.png">Mars
+                                    </a>
+                                </li>
+                                <li class="hidden-popup-list-item">
+                                    <a href="#" class="popup-item">
+                                        <img class="popup-item-img" alt="" src="img/blogs.png">Blogs
+                                    </a>
+                                </li>
+                                <li class="hidden-popup-list-item">
+                                    <a href="#" class="popup-item">
+                                        <img class="popup-item-img" alt="" src="img/promo-cut.png">Promos
+                                    </a>
+                                </li>
+                            </ul>
+                            <label for="more-apps-2" class="more-apps-2-label">Even more from Kottans</label>
+                            <input class="hidden-checkbox" type="checkbox" id="more-apps-2">
+                        </div>
+                    </div>
+                </li>
+
+                <li class="nav-menu-item">
+                    <a class="nav-menu-item-link" href="#"><img class="nav-image" src="img/bell.png" alt="bell"></a>
+                </li>
+                <li class="nav-menu-item">
+                    <a class="nav-menu-item-link" href="#"><img class="nav-image" src="img/iconfinder_user_light.png"
+                            alt="user-img"></a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+</body>
+
+</html>
```

Comment:
Add empty lines at the end of files. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

Reply:
and another stupid mistake, forgot that prettier deletes them

Reply:
Check the settings of your code editor he can add an empty line on save automatically)